linkis-public-enhancements/linkis-datasource/linkis-datasource-manager/server/src/main/java/org/apache/linkis/datasourcemanager/core/dao/mapper/DataSourceParamKeyMapper.xml

```diff
@@ -32,19 +32,42 @@
         <result property="refValue" column="ref_value"/>
         <result property="dataSource" column="data_source" />
     </resultMap>
+    <resultMap id="paramKeyMapEn" type="org.apache.linkis.datasourcemanager.common.domain.DataSourceParamKeyDefinition">
```

Comment:
Is it possible to use the same resultMap and use aliases for query fields?E.g：description_en as description, name_en as name